src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1166,13 +1166,9 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
-    winrt::hstring TerminalPage::_evaluatePathForCwd(const winrt::hstring& path)
+    std::wstring TerminalPage::_evaluatePathForCwd(const std::wstring_view path)
     {
-        const auto currentVirtualDir{ _WindowProperties.VirtualWorkingDirectory() };
-        const auto cwdString{ std::wstring_view{ currentVirtualDir } };
-        return winrt::hstring{
-            Utils::EvaluateStartingDirectory(cwdString, std::wstring_view{ path })
-        };
+        return Utils::EvaluateStartingDirectory(_WindowProperties.VirtualWorkingDirectory(), path);
     }
```

Comment:
`winrt::hstring` is implicitly convertible to `std::wstring_view`, so there's no need for explicit casts like this most of the time. It just doesn't work for something like `std::filesystem::path::operator/=`, because C++ wouldn't know which of the 31 overloads to call, given a `hstring`.

Also, when cppwinrt generates `String` parameters for functions, it'll not use `winrt::hstring` but `winrt::param::hstring` which binds to almost any string type, include `std::wstring`, without making any extra copies on the heap. We can thus just return a `std::wstring` from this function and it'll work without any additional modifications.